<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>brsgis_supp_pre_form</class>
 <widget class="QDialog" name="brsgis_supp_pre_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Supplemental Type</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose the type of supplemental you wish to create and click OK to continue:</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="supp_type">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>182</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContents</enum>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Plan by Others (P)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Document (D)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Raster (R)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Proposed / Design (P)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Historical Map / 1857 County (H)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tax Maps (T)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Map (M)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Folder (F)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other (O)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>USGS QUADS (Q)</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>brsgis_supp_pre_form</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>brsgis_supp_pre_form</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
